macros/safe_hash.sql

```diff
@@ -0,0 +1,9 @@
+{%- macro safe_hash(columns) -%}
+{% set coalesced_columns = [] %}
+{%- for column in columns -%}
+  {% do coalesced_columns.append("COALESCE(" ~ column ~ ", '')") %}
```

Comment:
should we also do lower/trim to the strings?

Reply:
good point - didn't consider this; better "real world" usage